### ISSUE TITLE
Vim increase <Ctrl-A> decreases end time

### ISSUE THREAD

**Issue by gerazov** [kristijanhusak/orgmode.nvim#95]
### Are you using "tree-sitter" branch?

No

### Describe the bug

Pressing `<C-a>` on the hours in the scheduled end time decreases them rather than increasing them. It does increase the minutes in the end time and the hours and minutes in the start time. 

### Steps to reproduce

1. Schedule an event with a time span.
2. Go to the hours in the end time.
3. Press `<C-a>` (or your remap).

### Expected behavior

I would expect the hours to increase.

### OS / Distro

Manjaro Linux

### Neovim version/commit

0.5

**Comment by kristijanhusak** [comment]
`<C-a>` by default should increase the date day by one. Did you change the default mappings?

**Comment by gerazov** [comment]
So `<C-a>` is overloaded in `orgmode`? I have `<C-a>` mapped as my tmux prefix so for the vim one I have:

```vim
nnoremap <a-x> <c-x>
nnoremap <a-a> <c-a>
vnoremap <a-x> <c-x>
vnoremap <a-a> <c-a>
nnoremap g<a-x> g<c-x>
nnoremap g<a-a> g<c-a>
vnoremap g<a-x> g<c-x>
vnoremap g<a-a> g<c-a>
```
So I guess the bug is related to the Vim <Ctrl-a>. This does increase the date (but doesn't update the day of week), and works reverse for the hours in the end of the period.

How can I remap the orgmode <Ctrl-a>?

**Comment by kristijanhusak** [comment]
You can check the docs here https://github.com/kristijanhusak/orgmode.nvim/blob/master/DOCS.md#org_show_help . This example shows how to map it to `+` and `-`.

**Comment by gerazov** [comment]
Ok great - this works well :+1:

Still, `<C-a>` on the hours of the end time decreases them ...

**Comment by kristijanhusak** [comment]
That's built in functionality, I cannot really control that.

**Comment by gerazov** [comment]
Ahhh :light: Vim sees this as a negative number because of the `-`

Yeah, an unfortunate coincidence  :sweat_smile:

**Comment by kristijanhusak** [comment]
@gerazov [tree-sitter](https://github.com/kristijanhusak/orgmode.nvim/tree/tree-sitter) branch now allows updating parts of the date, not only the whole day. See [org_timestamp_up](https://github.com/kristijanhusak/orgmode.nvim/blob/tree-sitter/DOCS.md#org_timestamp_up) and [org_timestamp_down](https://github.com/kristijanhusak/orgmode.nvim/blob/tree-sitter/DOCS.md#org_timestamp_down)

**Comment by gerazov** [comment]
Looks good :+1: 

I'm having a problem with it a bit. I have a messy init.vim so it's maybe just me :sweat_smile: but if you can reproduce I can repost this as a new issue:
- `org_timestamp_up` and `org_timestamp_down` both work only once for a given timestamp, i.e. it updates the year/month/day/time, but next use just brings the cursor on the following line. This doesn't happen for toggling between `<>` and `[]`, i.e. you can do this multiple times.
- incrementing/decrementing hours actually increments minutes by 5 min (this may be wanted behaviour, but it may be more useful to be able to increment/decrement hours and minutes separately)
- incrementing/decrementing starting time also increments/decrements end time (this may be wanted behaviour)
- incrementing/decrementing end time doesn't increment/decrement start time (this may be wanted behaviour)

**Comment by gerazov** [comment]
btw - is there a way to have `timestamp_up_day` work in agenda view? This would be supper useful for rescheduling tasks that were not done the previous day to today. Currently you have to `<Tab>` to them and increase them there.
